package.json

```diff
@@ -77,5 +78,9 @@
     "typedoc": "^0.20.33",
     "typescript": "^4.2.4",
     "uglifyify": "^5.0.2"
+  },
+  "jest": {
+    "verbose": true,
+    "testEnvironment": "jsdom"
```

Comment:
You were right, `testEnvironment` config here doesn't have any effect.

Reply:
Yeah, I'm keeping it and until we find an alternative.

Adding ` /* @jest-environment jsdom */` in the beginning of our test file seems to do the trick.